onnxruntime/test/providers/cpu/rnn/deep_cpu_lstm_op_test.cc

```diff
@@ -282,8 +273,7 @@ TEST(LSTMTest, MixedSequenceLengths) {
       1.3249796f, 1.51063104f, 1.61451544f};
 
   // Not able to mask on Y_c for CUDA using cudnn lib
```

Comment:
>// Not able to mask on Y_c for CUDA using cudnn lib [](start = 2, length = 51)

not needed?

Reply:
it's because this test sets the clip which makes it fall back to CPU. I'll create a follow up task to update the test to not add the clip attribute instead of add the 9999.9 if it's not set in the case. Also, there's a way to mask the data on Y_C, I'll fix it later.

---
In reply to: [238453395](https://github.com/Microsoft/onnxruntime/pull/77#discussion_r238453395) [](ancestors = 238453395)